govc/datastore/download.go

```diff
@@ -87,8 +107,8 @@ func (cmd *download) Run(ctx context.Context, f *flag.FlagSet) error {
 		return err
 	}
 
-	if cmd.OutputFlag.TTY {
-		logger := cmd.ProgressLogger("Downloading... ")
+	if cmd.DatastoreFlag.OutputFlag.TTY {
+		logger := cmd.DatastoreFlag.ProgressLogger(fmt.Sprintf("Downloading%s... ", via))
```

Comment:
Default message is just: "Downloading..."
If host is specified, looks like: "Downloading via esx1-host.example.com..."